### ISSUE TITLE
Accessibility Review: Tooltip

### ISSUE THREAD

**Issue by gakimball** [zurb/foundation-sites-6#17]
**Files:**
- [Documentation](https://github.com/zurb/foundation-sites-6/blob/master/docs/pages/tooltip.md)
- [Sass code](https://github.com/zurb/foundation-sites-6/blob/master/scss/components/_tooltip.scss)
- [JavaScript code](https://github.com/zurb/foundation-sites-6/blob/master/js/foundation.tooltip.js)

The tooltip plugin generates extra HTML. Each tooltip that's created gets a unique ID that can be used with `aria-describedby`. Here's what it looks like:

```html
<!-- This is the text that reveals the tooltip -->
<span data-tooltip="" aria-haspopup="true" class="has-tip" data-disable-hover="false" data-show-on="large" tabindex="1" title="" aria-describedby="s0pfg0" data-yeti-box="s0pfg0" data-toggle="s0pfg0">scarabaeus</span>

<!-- This is the tooltip text itself -->
<div class="tooltip" role="tooltip" aria-hidden="false" data-is-active="true" data-is-focus="false" id="s0pfg0" style="display: block; top: 591px; left: 223.938px; opacity: 1;">Fancy word for a beetle.</div>
```

For this, we need to make sure that AT will read the tooltip text description properly.

**Comment by zurbchris** [comment]
I've addressed most of these things with a new version of tooltip I've been working on. It's not yet feature complete, which is why I haven't merged it in / replaced the current tooltip. 
@joe-watkins the reason for using title, was so there could be a browser fallback if the JS failed to load. Though I have changed that to `data-tip-text="Thing to say here"` in the next version. Any feedback on that idea?

**Comment by christerFunka** [comment]
I can't see a reason to use aria-hidden in this case.
